ganttproject/src/net/sourceforge/ganttproject/search/SearchResult.java

```diff
@@ -19,28 +19,51 @@
 package net.sourceforge.ganttproject.search;
 
 public class SearchResult<SearchObjectType> {
+  private final int id;
   private final String myOrigin;
-  private final String mySnippet;
-  private final String myLabel;
+  private final String secondaryLabel;
```

Comment:
May I ask you to change the names according to the surrounding style? We use prefix `my` for member variables and do not use `this`.